pull_request_template.md

```diff
@@ -1,18 +1,9 @@
-## What did you do? 
-- [describe the proposed changes]
-- 
-
-
-## Why did you make this change?
-(Cite Issue number OR provide rationalization of changes if no issue exists)
-(If fixing a bug, please add any relevant error or traceback)
-
+(Please replace this header with a description of your pull request. Please include *BOTH* what you did and why you made the change. The "why" may simply be citing a relevant Galaxy issue.) 
 
 ## How to test the changes? 
-(select the most appropriate option; if the latter, provide steps for testing below)
 - [ ] I've included appropriate [automated tests](https://docs.galaxyproject.org/en/latest/dev/writing_tests.html).
```

Comment:
I'd keep the guidance, from the discussion at today's Dev Meeting it seemed it was not clear the author was supposed to choose one option.

```suggestion
(Select one of the following 3 options)
- [ ] I've included appropriate [automated tests](https://docs.galaxyproject.org/en/latest/dev/writing_tests.html).
```

Reply:
But it could be more than one of these three options right - I've checked multiple ones before? I'll redo it with this language though. I guess the important change is to make it clear if there first two are checked we don't need the third.